json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -233,6 +241,32 @@ protected function result() {
 		);
 	}
 
+	public $items = [];
+
+	public function jetpack_sync_send_data_listener() {
+		foreach ( func_get_args()[0] as $key => $item ) {
```

Comment:
I don't understand what this does. Since this function doesn't return anything...

Reply:
it stores actions into `$this->items` that we then send.

Reply:
🤦‍♂